<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>501</width>
    <height>322</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>150</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>501</width>
    <height>322</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Email Sender</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
    background-color: #222222;
    color: #ffffff;
    font-family: &quot;Roboto&quot;, sans-serif;
    font-size: 14px;
}

QLabel {
    color: #ffffff;
}

QPushButton {
    background-color: #333333;
    color: #ffffff;
    border-radius: 5px;
    padding: 10px;
}

QPushButton:hover {
    background-color: #444444;
}

QLineEdit {
    background-color: #333333;
    color: #ffffff;
    border: 1px solid #444444;
    padding: 10px;
}

QTextEdit {
    background-color: #333333;
    color: #ffffff;
    border: 1px solid #444444;
    padding: 10px;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>3</number>
      </property>
      <widget class="QWidget" name="pageSingleOrBatch">
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="QPushButton" name="pushButtonSingle">
          <property name="text">
           <string>Single</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="pushButtonBatch">
          <property name="maximumSize">
           <size>
            <width>200</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Batch</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="pageUploadFile">
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <widget class="QPushButton" name="pushButtonUpload">
          <property name="maximumSize">
           <size>
            <width>200</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Upload .csv file</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="labelFileName">
          <property name="maximumSize">
           <size>
            <width>300</width>
            <height>20</height>
           </size>
          </property>
          <property name="text">
           <string>No file selected</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="pushButtonCompose">
          <property name="maximumSize">
           <size>
            <width>200</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Compose</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="pageComposeSingle">
       <widget class="QLineEdit" name="lineEditBody">
        <property name="geometry">
         <rect>
          <x>-5</x>
          <y>94</y>
          <width>483</width>
          <height>170</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>170</height>
         </size>
        </property>
        <property name="readOnly">
         <bool>false</bool>
        </property>
        <property name="placeholderText">
         <string>Body</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButtonSend">
        <property name="geometry">
         <rect>
          <x>-5</x>
          <y>270</y>
          <width>239</width>
          <height>36</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>40</height>
         </size>
        </property>
        <property name="text">
         <string>Send</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEditEmail">
        <property name="geometry">
         <rect>
          <x>-5</x>
          <y>2</y>
          <width>483</width>
          <height>40</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <property name="placeholderText">
         <string>Email</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEditSubject">
        <property name="geometry">
         <rect>
          <x>-5</x>
          <y>48</y>
          <width>483</width>
          <height>40</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <property name="placeholderText">
         <string>Subject</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButtonReset">
        <property name="geometry">
         <rect>
          <x>240</x>
          <y>270</y>
          <width>238</width>
          <height>36</height>
         </rect>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>40</height>
         </size>
        </property>
        <property name="text">
         <string>Reset</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="pageSuccessfullySent">
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0" colspan="2">
         <widget class="QLabel" name="labelEmailSent">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>40</height>
           </size>
          </property>
          <property name="text">
           <string>Email sent</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="pushButtonExit">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Close</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="pushButtonNew">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>New</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
